Tests/Unit/Mail/MailerHelperTest.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+
+namespace Sulu\Bundle\FormBundle\Tests\Unit\Mail;
+
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Sulu\Bundle\FormBundle\Mail\MailerHelper;
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Mime\Address;
+
+/**
+ * Class MailerHelperTest
+ * This tests the MailerHelper class that sends an email using the symfony mailer component.
+ *
+ * The MailerHelper class has 3 protected methods that could be overwritten to change the mail message.
+ * The setHeaders must set the headers of the email including to cc bcc ect.
+ * The setBody must set the html or text body of the email.
+ * The setAttachments should set attachments to the email.
+ * The setAttachments should set attachments to the email.
+ * in the future it might be nice to delegate the methods to a separate service.
+ *
+ * @package Sulu\Bundle\FormBundle\Tests\Unit\Mail
+ */
+class MailerHelperTest extends TestCase
+{
+    /**
+     * @var MailerHelper
+     */
+    private $mailerHelper;
+
+    /**
+     * @var MockObject|MailerInterface
+     */
+    private $mailerMock;
+
+    /**
+     * @var MockObject|LoggerInterface
+     */
+    private $loggerMock;
+
+    protected function setUp(): void
+    {
+        $this->mailerMock = $this->getMockBuilder(MailerInterface::class)->getMock();
```

Comment:
We use `prophecy` to for mocking so I would keep that. Ping me or contact me on slack if you need any help.